tests/wandb_config_test.py

```diff
@@ -28,3 +30,14 @@ def test_locked_set():
     s.update_locked(dict(this=2, that=4), "sweep")
     s.this = 8
     assert s.this == 2
+
+
+def test_load_config_default():
+    test_path = os.path.join("config-defaults.yaml")
```

Comment:
Nit pick: Here `os.path.join()` is a No-op. `test_path = "config-defaults.yaml"` would do.